.github/workflows/ci.yml

```
@@ -10,26 +10,34 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
-          - '1.9'
-          - 'nightly'
+          - '1'    # Current stable version
```

I think it would be better to keep tests on LTS.
```suggestion
          - '1.6'  # LTS
          - '1'    # Current stable version
```